data/effects.txt

```diff
@@ -216,3 +216,63 @@ effect "bolide"
 	"random spin" 5
 	"velocity scale" .8
 	"random frame rate" 6
+	
+effect "pug tiny explosion"
+	sprite "effect/explosion/pug tiny"
```

Comment:
If you change the location of the explosions to be in their own folder, these would need to change to `sprite "effect/explosion/pug/<size>`.